pkg/controller/actuator.go

```diff
@@ -165,15 +193,26 @@ func (a *actuator) Reconcile(ctx context.Context, log logr.Logger, ex *extension
 		hosts,
 		shootSpecificCIDRs,
 		alwaysAllowedCIDRs,
+		istioNamespace,
+		istioLabels,
```

Comment:
Does this make sense if we have them stored in the Actuator "interface"? If we have to pass them along every time.

Reply:
I rather have them being passed around explicitly. Otherwise the next thing you know is I build another "kind of global state" that bites us later. Last thing we want is that some ACL rules are assigned to wrong Ingresses because the struct fields haven't been properly resetted 🙃